Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM readytalk/nodejs
 
 # Add our configuration files and scripts
 WORKDIR /app
-ADD . /app
+COPY . /app
```

Comment:
Using COPY rather than ADD. ADD is needed if we have set sym links and such, for what I've checked we don't have, so it's recommended to use COPY instead.